src/tests/tribol_iso_integ.cpp

```diff
@@ -44,34 +41,21 @@ class IsoIntegTest : public ::testing::Test {
 
   bool integrate( RealT const tol )
   {
-    RealT xyz[this->dim * this->numNodes];
-    RealT* xy = xyz;
-
-    RealT* x = this->x;
-    RealT* y = this->y;
-    RealT* z = this->z;
+    // axom::primal::ZipIndexable<axom::primal::Vector<RealT, dim>> xyz;
```

Comment:
do you need to keep this primal line?

Reply:
Nope, just did a test for the syntax at some point and forgot to remove.  Fixed